src/include/verbose.h

```diff
@@ -109,6 +109,18 @@ struct __wt_verbose_multi_category {
 /* Check if a given verbosity level satisfies the verbosity level of a category. */
 #define WT_VERBOSE_LEVEL_ISSET(session, category, level) (level <= S2C(session)->verbose[category])
 
+/* Check if a given verbosity level is met for eviction related messages. */
+#define WT_VERBOSE_EVICT_LEVEL_ISSET(session, level)         \
+    (S2C(session)->verbose[WT_VERB_EVICT] >= level &&        \
+      S2C(session)->verbose[WT_VERB_EVICT_STUCK] >= level && \
+      S2C(session)->verbose[WT_VERB_EVICTSERVER] >= level)
+
+/* Set the verbosity level for eviction related messages. */
+#define WT_SET_EVICT_VERBOSE_LEVEL(session, level)      \
```

Comment:
We can directly set these without a #define, since these macros are not going to be used anywhere else.